docs/user/upgrading_renku.rst

```diff
@@ -0,0 +1,89 @@
+.. upgrading_renku:
+
+Upgrading a Renku Project
```

Comment:
I think here you're referring to a user's renku project, but in the line below you mean the Renku Project as a whole - maybe call it something different here? or say "Keeping your project up-to-date"

Reply:
yeah good point! fixing.

Reply:
I don't think this is fixed?

Reply:
I fixed the lines below it to just refer to user's renku project.

Reply:
hmm this is still confusing - this is what it looks like:

![image](https://user-images.githubusercontent.com/3353586/83623190-4bdf7c80-a591-11ea-91c7-cf1b698821ef.png)

"Renku Project" is used to mean two different things. Maybe we can call this section "Upgrading the image in your project"? Because it's not just upgrading the CLI it's also useful to know how to upgrade the image for a runtime update.